playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms.j2

```diff
@@ -294,7 +294,6 @@ location ~ ^{{ EDXAPP_MEDIA_URL }}/(?P<file>.*) {
 
   {% include "robots.j2" %}
   {% include "static-files.j2" %}
-  {% include "shib-only.j2" %}
```

Comment:
Why are we removing this include? `shib-only.j2` still exists and has stuff in it?

Reply:
The redirects in that file only apply to cms now